### PR TITLE
Try to bring coverall test coverage comments back

### PR DESCRIPTION
**Pull request by kezhenxu94**
Please answer these questions before submitting pull request

- Why submit this pull request?
- [ ] Bug fix
- [ ] New feature provided
- [ ] Improve performance

- Related issues

___
### Bug fix
- Bug description.

- How to fix?

___
### New feature or improvement
- Describe the details and related test reports.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,6 +12,7 @@ microservices, cloud native and container-based (Docker, Kubernetes, Mesos) arch
 
 [![Maven Central](https://img.shields.io/maven-central/v/org.apache.skywalking/apache-skywalking-apm.svg)](http://skywalking.apache.org/downloads/)
 [![Build Status](https://travis-ci.org/apache/skywalking.svg?branch=master)](https://travis-ci.org/apache/skywalking)
+[![Coverage Status](https://coveralls.io/repos/github/apache/skywalking/badge.svg?branch=master)](https://coveralls.io/github/apache/skywalking?branch=master)
```
Comment (wu-sheng) [inline README.md]: Show 17% coverage is not cool :p

**Comment by wu-sheng** [comment]
I have deleted the old repo on https://coveralls.io, asking the INFRA to set up a new one. You could follow this, https://issues.apache.org/jira/browse/INFRA-19135

**Comment by kezhenxu94** [comment]
Seems we're making progress now 
image

**Comment by wu-sheng** [comment]
Finally, but still 7%? Do we set the codebase right? Is test folder included?

**Pull request merged**